env0/resource_template_project_assignment_test.go

```diff
@@ -0,0 +1,108 @@
+package env0
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/golang/mock/gomock"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestUnitTemplateProjectAssignmentResource(t *testing.T) {
```

Comment:
add assertion when `apiClient.AssignTemplateToProject` throw error
and for the Read function where the template have multiple project ids

Reply:
for the read function I can  add another id to the variable `returnValues,` and it will check it in the exist test "create"

Reply:
@samuel-br after last talk can you give the commit where you fix this issue?

Reply:
cc793cb3b075099bc299bdb5196d97d4fce423d8